src/test/java/teammates/test/cases/logic/FeedbackResponseCommentsLogicTest.java

```diff
@@ -264,40 +265,33 @@ public void testUpdateFeedbackResponseComment() throws Exception {
     }
 
     @Test
-    public void testDeleteFeedbackResponseComment() throws Exception {
+    public void testDeleteFeedbackResponseCommentById() throws Exception {
         //create a frComment to delete
         FeedbackResponseCommentAttributes frComment = restoreFrCommentFromDataBundle("comment1FromT1C1ToR1Q1S1C1");
```

Comment:
`//create a frComment to delete` The comment is not valid any more.

In addition, move the variable declaration to the place where it is first used.

Reply:
@xpdavid which variable you are talking about?

Reply:
`frComment `

Reply:
Made the change.